examples/dapperish.go

```diff
@@ -12,7 +12,9 @@ import (
 	"runtime"
 	"strings"
 
-	"github.com/opentracing/api-golang/examples/dapperish"
+	// `go imports` was stumbling over dapperish since it's both a package name
+	// and a source file.
+	dapperish "github.com/opentracing/api-golang/examples/dapperish"
```

Comment:
yeah, I will try to upgrade my vim stuff and see if that fixes it. out of curiosity I ran `goimports` directly and it worked fine.